tests/e2e/tidbcluster/across-kubernetes.go

```diff
@@ -212,6 +212,22 @@ var _ = ginkgo.Describe("[Across Kubernetes]", func() {
 			framework.ExpectNoError(err, "tc1 is not connectable")
 		})
 
+		ginkgo.It("Deploy clusters and cluster-2 do not have PD", func() {
```

Comment:
Please also help cover all the components in the heterogeneous cases in tidbcluster.go.